Rest API - Delete User - Refactor #147

n-machiavelli opened the pull request:
Rest API - Delete User - Refactor #147

n-machiavelli committed:
Rest API - Delete User - Refactor #147

Rest API - Delete User - Refactor #147

Code4SocialGood commented on src/main/java/org/c4sg/controller/UserController.java:
```diff
@@ -141,7 +144,11 @@ public void deleteUser(@ApiParam(value = "User id to delete", required = true)
         try {
             userService.deleteUser(id);
-        } catch (Exception e) {
+            if (userService.findById(id) !=null){
```
1. Should this be organizationService.findByUserId(userId)?
2. Move the logic of deleting org to UserService. Deleting Org is within the business logic of deleting user.

n-machiavelli committed:
Review - Rest API - Delete User - Refactor #147

1. Should this be organizationService.findByUserId(userId)?
2. Move the logic of deleting org to UserService. Deleting Org is within
the business logic of deleting user.
